app/scripts/controllers/workflowdetails.js

```diff
@@ -154,6 +154,9 @@ angular.module('dockstore.ui')
 
       $scope.getDaysAgoString = function(timestamp) {
         var daysAgo = $scope.getDaysAgo(timestamp);
+        if(daysAgo < 0){
```

Comment:
I think this was added to this branch by accident. Leave it for now, but in the future it is best to make feature branches independent unless it is necessary to build one feature on top of another.